### PR TITLE
Added DE_CH Keyboard Layout

### PR DESCRIPTION
**Pull request by magbeat**

### CODE REVIEW

**Comment by aioutecism** [comment]
450px-kb_swiss svg

File: src/Layouts/DE_CH_QWERTZ.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {LayoutMap} from './Layout';
+
+export const id = 'DE_CH QWERTY';
```
Comment (aioutecism) [inline src/Layouts/DE_CH_QWERTZ.ts]: Should this be `DE_CH QWERTZ`?

Comment (magbeat) [inline src/Layouts/DE_CH_QWERTZ.ts]: You're right of course. Sorry. Pushed again.

**Comment by aioutecism** [comment]
Great!

**Pull request merged**